src/core/include/units/quantity.h

```diff
@@ -139,7 +139,9 @@ class quantity {
 
   template<typename Value>
     requires(safe_convertible_to_<std::remove_cvref_t<Value>, rep>)
-  constexpr explicit(!detail::quantity_one<quantity>) quantity(Value&& v) : number_(std::forward<Value>(v))
+  constexpr explicit(!detail::quantity_one<quantity>)
+    quantity(Value&& v) noexcept(std::is_nothrow_constructible_v<Rep, Value&&>) :
```

Comment:
The `std::declval` it uses adds `&&` for you.
```suggestion
    quantity(Value&& v) noexcept(std::is_nothrow_constructible_v<Rep, Value>) :
```

Reply:
Done.